src/components/EditorsColumn.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import includes from 'lodash/includes';
+import partial from 'lodash/partial';
+
+import EditorContainer from './EditorContainer';
+import Editor from './Editor';
+
+function allErrorsFor(language, errors, runtimeErrors) {
+  if (language === 'javascript') {
+    return errors.javascript.items.concat(runtimeErrors);
+  }
+  return errors[language].items;
+}
+
+export default function EditorsColumn({
+  currentProject,
+  errors,
+  onComponentMinimize,
+  onEditorInput,
+  onRequestedLineFocused,
+  runtimeErrors,
+  ui,
+}) {
+  const editors = [];
+  ['html', 'css', 'javascript'].forEach((language) => {
+    if (includes(ui.minimizedComponents, `editor.${language}`)) {
+      return;
+    }
+
+    editors.push(
+      <EditorContainer
+        key={language}
+        language={language}
+        source={currentProject.sources[language]}
+        onMinimize={
+          partial(onComponentMinimize, `editor.${language}`)
+        }
+      >
+        <Editor
+          errors={allErrorsFor(language, errors, runtimeErrors)}
+          key={language}
+          language={language}
+          percentageOfHeight={1 / editors.length}
```

Comment:
Oh I meant to comment on the bug issue for that, my resize branch actually fixes that issue (by explicitly demanding resizing on min/maxing).